webapp-django/staticfiles/report-index/page-heading/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class PageHeading extends React.Component {
+  render() {
+    return (
+      <div className="page-heading">
+        <h2>
+          {this.props.product} {this.props.version} Crash Report [@ {this.props.signature} ]
+        </h2>
+      </div>
+    );
+  }
+}
+
+PageHeading.propTypes = {
```

Comment:
Whoops, that's for static methods, not static properties. That's still just a proposal, although there's a Babel rule for it.

Truth be told I'm not a huge fan of propTypes because they're checked at runtime in development mode only. Lemme think about this.

Reply:
For now leave this as is.